src/main/java/com/google/api/generator/engine/ast/StringObjectValue.java

```diff
@@ -37,6 +37,12 @@ public TypeNode type() {
   public abstract static class Builder {
     public abstract Builder setValue(String value);
 
-    public abstract StringObjectValue build();
+    public abstract StringObjectValue autobuild();
+
+    public StringObjectValue build() {
+      String value = String.format("%s%s%s", "\"", JavaEscaper.escape(autobuild().value()), "\"");
```

Comment:
Can we avoid calling `autoBuild` twice?

Reply:
I think calling autoBuild twice is necessary, the first one is for creating a stringObjectValue, then we use escaper to handle special characters, a new string value is to be set. since if the object is built we cannot re-set the value (autovalue pattern) unless we re-create one (second autoBuild).
if we set the value by below to avoid the second `autoBuild`, there will be OutOfMemoryError, coz it will be a loop `build() -> setvalue() -> build()` 
```
Builder builder= setValue(value);
return builder.build();
```